controllers/eip_association_controller.go

```diff
@@ -0,0 +1,80 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	awsv1alpha1 "github.com/logmein/k8s-aws-operator/api/v1alpha1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// EIPReconciler reconciles a EIP object
+type EIPAssociationReconciler struct {
+	client.Client
+	Log logr.Logger
+}
+
+// +kubebuilder:rbac:groups=aws.k8s.logmein.com,resources=eipassociations,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=aws.k8s.logmein.com,resources=eipassociations/status,verbs=get;update;patch
```

Comment:
You didn't add a status subresource. Should we add one to show the assignment status?

Reply:
Nope removed the rbac rule